### ISSUE TITLE
Syncing files failed on Apple M1

### ISSUE THREAD

**Issue by wxw-matt** [loft-sh/devspace#1493]
**What happened?**  
Let the error messages go first:
```
E0710 01:51:02.257212   41773 v2.go:151] io: read/write on closed pipe
Sync Error on /home/wxw/docker/devspace-quickstart-nodejs: Sync - connection lost to pod my-namespace/devspace-quickstart-nodejs-8589bb75b8-njmnh-devspace:  command terminated with exit code 126
```
I followed the official [tutorial](https://devspace.sh/cli/docs/getting-started/installation) to set up Devspace.
Everything worked well but this command: `devspace dev`, which reported the errors I posted.

**What did you expect to happen instead?**  
It should sync the files between the container (pods) and local path.

**How can we reproduce the bug?** (as minimally and precisely as possible)  
Just run `devspace dev`

**Local Environment:**  
- DevSpace Version: devspace version 5.14.3
- Operating System: macOS (Apple M1)
- Deployment method: helm

**Kubernetes Cluster:**  
- Cloud Provider: minikube
- Kubernetes Version: GitVersion:"v1.20.7"

**Anything else we need to know?**  
Output from `kubectl get pods`
```
NAME                                                   READY   STATUS    RESTARTS   AGE
devspace-quickstart-nodejs-8589bb75b8-njmnh-devspace   1/1     Running   0          20m
```
Output from `kubectl get services`
```
NAME                         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
devspace-quickstart-nodejs   ClusterIP   10.101.50.228   <none>        3000/TCP   22m
```
Complete logs
```
[info]   Using namespace 'my-namespace'
[info]   Using kube context 'minikube'
[info]   Waiting for pods...
[info]   Starting sync...
[info]   Inject devspacehelper into pod my-namespace/devspace-quickstart-nodejs-8589bb75b8-njmnh-devspace
[info]   Start syncing
[done] √ Sync started on /home/wxw/docker/devspace-quickstart-nodejs <-> . (Pod: my-namespace/devspace-quickstart-nodejs-8589bb75b8-njmnh-devspace)
[error]  Sync Error on /home/wxw/docker/devspace-quickstart-nodejs: Sync - connection lost to pod my-namespace/devspace-quickstart-nodejs-8589bb75b8-njmnh-devspace:  command terminated with exit code 126
[info]   Sync stopped
[info]   Restarting sync...
[info]   Waiting for pods...
[info]   Starting sync...
E0710 02:10:10.146895   47118 v2.go:151] io: read/write on closed pipe
[info]   Inject devspacehelper into pod my-namespace/devspace-quickstart-nodejs-8589bb75b8-njmnh-devspace
[info]   Start syncing
[done] √ Sync started on /home/wxw/docker/devspace-quickstart-nodejs <-> . (Pod: my-namespace/devspace-quickstart-nodejs-8589bb75b8-njmnh-devspace)
[error]  Sync Error on /home/wxw/docker/devspace-quickstart-nodejs: Sync - connection lost to pod my-namespace/devspace-quickstart-nodejs-8589bb75b8-njmnh-devspace:  command terminated with exit code 126
[info]   Sync stopped
```

/kind bug

**Comment by FabianKramm** [comment]
@wxw-matt thanks for creating this issue! Is this closed through #1495 ?

**Comment by wxw-matt** [comment]
> @wxw-matt thanks for creating this issue! Is this closed through #1495 ?

Yup, this issue has been fixed as well
